pkg/network/network.go

```diff
@@ -24,25 +24,25 @@ type NetworkInfo struct {
 
 // retrieve network info from endpoint
 func GetNetworkInfo() (*NetworkInfo, error) {
-	if os.Getenv("STANDALONE") == "1" {
-		return &NetworkInfo{Network: defaultNetwork, URL: defaultNodeUrl}, nil
-	}
-
 	resp, err := http.Get(massaStationNodeEndpoint)
 	if err != nil {
-		return nil, fmt.Errorf("failed to make GET request: %v", err)
+		log.Warnf(fallbackMsg+"Failed to make GET request: %v", err)
```

Comment:
```suggestion
logFallback("GET massa station node endpoint", err)
```

With:
```golang
func logFallback(action string, err error) {
     log.Warnf("Fallback to default network: failed to %s: %v", msg, err)
 }

Reply:
I did the fix here https://github.com/massalabs/station-massa-wallet/pull/530/commits/688a2a5b0b1f1fa50fe07fbd5a711f65509ed9a2